ui/src/authors/containers/AuthorCitationsContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useMemo } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import LiteratureSearchContainer from '../../literature/containers/LiteratureSearchContainer';
+import { AUTHOR_CITATIONS_NS } from '../../reducers/search';
+
+function AuthorCitations({ authorBai }) {
+  const baseQuery = useMemo(
+    () => ({
+      q: `refersto a ${authorBai}`,
+    }),
+    [authorBai]
+  );
+
+  return (
+    <LiteratureSearchContainer
+      namespace={AUTHOR_CITATIONS_NS}
+      baseQuery={baseQuery}
```

Comment:
don't we need `baseAggregationsQuery` too, if we have to author facet, we need to exclude that name right?

Reply:
No, in this tab we don't need to exclude the author in any facet